locales-pending/landing-premium.ftl

```diff
@@ -103,5 +103,11 @@ landing-premium-plans-cards-feature-guidance = <b>Guided help</b> to fix high-ri
 landing-premium-plans-cards-feature-monitoring = Continuous monitoring
 
 # Value proposition
+
+landing-premium-value-prop-fix-exposures-description = We provide steps to follow when you’ve been affected by a data breach and can even remove your data from more than 190 sites trying to sell it — and we <privacy_link>respect your privacy</privacy_link> in the process.
```

Comment:
Don't we use a variable for the number `190` elsewhere, since the number might go up in the future? (In which case this would also become a pluralised string.)